core/src/Revolution/Registry/Db/sqlsrv/modDbRegisterMessage.php

```diff
@@ -25,171 +25,171 @@ class modDbRegisterMessage extends \MODX\Revolution\Registry\Db\modDbRegisterMes
             'payload' => NULL,
             'kill' => 0,
         ),
-        'fieldMeta' =>
+        'fieldMeta' => 
```

Comment:
Are those spaces generated by xPDO's parse schema script?

Reply:
Most likely, I didn't touch any of these files

Reply:
Yeah, this is how var_export always does this. It appears at some point someone edited these files by hand and their IDE likely trimmed the whitespace automatically (or they ran it through some kind of code formatting).

Reply:
After adding the editorconfig linter, there will be no such cases.